src/pages/parametrer-hymne/1.tsx

```diff
@@ -85,6 +85,7 @@ const AnthemStep1 = () => {
           <div style={{ display: 'flex', flexDirection: 'column' }}>
             <p style={{ margin: '25px 0 25px' }}>La piste vocal du couplet, La La.</p>
             {data &&
+              data.verseAudios.length > 0 &&
               data.verseAudios.map((audio, idx) => (
```

Comment:
If `verseAudios ` is undefined, `data.verseAudios.length` will crash too. Trying to get length from undefined.
The solution is to use a default value: `(data.verseAudios || []).map`.

I'll update master and close this PR.